caps/templates/includes/council-powers.html

```diff
@@ -0,0 +1,135 @@
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>embedding carbon impact assessment as part of the council’s budgeting and financial accountability process</li>
+                <li>utilising Public Works Loan Board loans or the Business Rates Retention Scheme to invest in emissions-reducing capital projects that otherwise wouldn’t get funded</li>
+                <li>specifying low carbon equipment and practices when procuring for relevant services from suppliers</li>
+                <li>prioritising positive environmental impacts during procurement, through the Public Services (Social Value) Act 2012</li>
+            </ul>
+        </div>
+    </details>
+
+{% if council.authority_type != 'COMB' and council.authority_type != 'NMD' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Transport planning</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>incentivising ‘Active Travel’ like walking and cycling by creating or widening footways and cycleways</li>
+                <li>incentivising Electric Vehicle use by assigning street space to EV charging</li>
+                <li>disincentivising the use of traditional cars through congestion charging or the reduction of parking space</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type == 'UA' or council.authority_type == 'MD' or council.authority_type == 'CTY' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Passenger transport</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>encouraging bus and private hire operators to use low-emission vehicles, through licensing requirements or Clean Air Zones</li>
+                <li>incentivising bus use by improving routes, timetables, and ticket prices through ‘Enhanced Partnerships’ with operators</li>
+                <li>proposing to central government the development of light rail / tram networks that integrate, rather than compete, with other modes of transport in the area</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type != 'COMB' and council.authority_type != 'NMD' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Schools and libraries</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>reducing the carbon footprint of civic buildings through better insulation and renewable energy use</li>
+                <li>incentivising ‘Active Travel’ or public transport use among employees</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type != 'COMB' and council.authority_type != 'CTY' %}
```

Comment:
Could combine these two ifs

Reply:
I’ve restructured the template to do less fancy logic, so this isn’t an issue any more.